chapter07/mdw/Fred.txt

```diff
@@ -0,0 +1,18 @@
+Mr. Slate had an employee
+Fred Filntstone was his name
+fred was happy
+fred was married to wilma
+wilma and betty were friends
+fred was the father of bammbamm
+Alfred yelled for BATMAN.
+barney yelled for FRED
+nobody was in this line
+frederick was alone
+so was sam
+and pat
+writer has A breakdown
+wilma is fred?
+freD?
+fRedom!!!
+FLAGRAGa!
+FLLLlRRRG
```

Comment:
_**flushing out what she just read and restoring her childhood memories**_  :expressionless: